src/alertsmanagement/README.md

```diff
@@ -0,0 +1,46 @@
+# Azure CLI alertsmanagement Extension
+
+This extension can manage alert related resources. Currently, it supports
+action rule management.
+
+### How to use
+Install this extension using the below CLI command
+```
+az extension add --name alertsmanagement
+```
+
+### Action rule
+Action rule documentation: https://docs.microsoft.com/en-us/azure/azure-monitor/platform/alerts-action-rules.
+
+Create an action rule to suppress notifications for all Sev4 alerts on all VMs within the subscription every weekend.
+```
+az monitor action-rule create --resource-group rg --name rule --location Global --status Enabled --rule-type Suppression --severity Equals Sev4 --target-resource-type Equals Microsoft.Compute/VirtualMachines --suppression-recurrence-type Weekly --suppression-recurrence 0 6 --suppression-start-date 12/09/2018 --suppression-end-date 12/18/2018 --suppression-start-time 06:00:00 --suppression-end-time 14:00:00
```

Comment:
can we do a nice convenient layer to map the int value to day of week?

Reply:
Do you mean map enum to int?

Reply:
I mean map int to day of week. is there existing practice in CLI to specify day of week?

Reply:
if there's no good way to do it, at least clear help message on how to specify day of week.

Reply:
Yes. `--suppression-recurrence Sunday Saturday` will be more friendly to users.
This parameter supports both day of week and day of month.